src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package Pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+//    By avatarIcon = By.cssSelector("img.avatar");
+
+    @FindBy(css = "img[class='avatar']")
+    WebElement userAvatarIcon;
+
+    public boolean getUserAvatarIcon(){
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(2));
+        return wait.until(d->userAvatarIcon.isDisplayed());
+//        return userAvatarIcon.isDisplayed();
+    }
+
+//    public boolean  getUserAvatarIcon(){
+//        return findElementUsingByLocator(avatarIcon).isDisplayed();
+//    }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.